### DISCUSSION TITLE
Console under nginx reverse proxy

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk#5813] [Feature Request]
The correct way to set nginx as a reverse proxy is as follows

```
        location / {
           proxy_set_header        X-Real-IP       $remote_addr;
           proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;

           proxy_pass http://127.0.0.1:21114/;
           }

```

Otherwise, in the list of connected devices, the IP address is 127.0.0.1

This was before the change

![image](https://github.com/rustdesk/rustdesk/assets/8618744/69442dda-efc6-4a43-9b7e-9339d26bb02a)

and this after

![image](https://github.com/rustdesk/rustdesk/assets/8618744/50017286-2cad-455d-98e6-7df057e5dad9)

The installation script should be updated

Regards
Pasquale

**Comment by pdifeo** [comment]
Sorry, wrong discussion. Is for rustdesk-server-pro

**Reply by dinger1986** [reply]
let me test and get it added
